wifiphisher/common/phishinghttp.py

```diff
@@ -157,6 +165,12 @@ def post(self):
         file_path = template_directory + render_file
         self.render(file_path, **template.get_context())
 
+        # Finds victim object that corresponds to ip address and try to
+        # Discover OS by request
+        victims_instance = victim.Victims.getInstance()
```

Comment:
make the code more consistent: Instead of camel syntax change to the underline style: `get_instance`